lib/cache/cache.js

```diff
@@ -0,0 +1,20 @@
+var cacheManager = require('cache-manager')
+
+class Cache {
+  constructor () {
+    this.cache = null
+
+    switch (process.env.CACHE_STORAGE) {
+      case 'redis':
+        var redisStore = require('cache-manager-ioredis')
+        this.cache = cacheManager.caching({ store: redisStore, url: process.env.REDIS_URL, refreshThreshold: 3 })
+        break
+      default:
+        this.cache = cacheManager.caching({ store: 'memory', max: 100, ttl: 10 })
```

Comment:
what happens if there are more than 100 repos, what happens then? is it only storing the last 100 repo settings?

Reply:
hey @shine2lay,

Check the stackoverflow answer: https://stackoverflow.com/a/24230894